src/components/TagManager.jsx

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Head from 'next/head';
 import { getTrackingDetails } from 'utils/tracking';
+import { AccountId } from 'utils/deploymentInfo';
+import nextConfig from 'next/config';
 
 const TagManager = ({ environment }) => {
   const { enabled, containerId } = getTrackingDetails(environment);
-
+  const accountId = nextConfig()?.publicRuntimeConfig?.accountId;
   // if tracking is not enabled don't add tag manager to the head
   if (!enabled) return (null);
 
+  // whether to use biomage or HMS matomo account
+  const matomoName = accountId === AccountId.BIOMAGE ? 'biomage' : 'cellenics';
```

Comment:
I prefer not to have organization names appearing in the code base more than absolutely necessary and also to make it so that it would be easy for another party to join. We could add this to a config and and here it could just be something like:

 `const matomoName = matomoNameByAccountId[accountId]`

Reply:
Thanks! looks good